CHANGELOG.md

```diff
@@ -1,19 +1,24 @@
-- Fixed the ability to change default search via preferences ([#4695](https://github.com/brave/browser-laptop/issues/4695))([#4696](https://github.com/brave/browser-laptop/issues/4696))
-- Fixed Brave Wallet showing coinbase as an option for unsupported countries ([#4615](https://github.com/brave/browser-laptop/issues/4615))
-- Fixed double clicking bookmark in bookmark manager opening many many tabs ([#4523](https://github.com/brave/browser-laptop/issues/4523))
-- Fixed toggling Bookmarks Toolbar from "Bookmarks" menu ([#3356](https://github.com/brave/browser-laptop/issues/3356))
-- Fixed bookmarks toolbar "hover after click" behavior on folders ([#4361](https://github.com/brave/browser-laptop/issues/4361))
-- Fixed an edge case of multiple Payment History receipts being generated on the same day ([#4632](https://github.com/brave/browser-laptop/issues/4632))([#4605](https://github.com/brave/browser-laptop/issues/4605))
-- Fixed Windows titlebar to ignore right alt key (AltGr) ([#4628](https://github.com/brave/browser-laptop/issues/4628))([#4626](https://github.com/brave/browser-laptop/issues/4626))
+- Added Pocket extension ([#4687](https://github.com/brave/browser-laptop/issues/4687))([#4692](https://github.com/brave/browser-laptop/issues/4692))([#4699](https://github.com/brave/browser-laptop/issues/4699))
+- Added Brave Wallet backup and recovery ([#4743](https://github.com/brave/browser-laptop/issues/4743))([#3350](https://github.com/brave/browser-laptop/issues/3350))
+- Added a way for users to opt-out of crash reporting ([#4479](https://github.com/brave/browser-laptop/issues/4479))
+- Added multi-select deletion and drag and drop in about:history ([#3949](https://github.com/brave/browser-laptop/issues/3949))([#4741](https://github.com/brave/browser-laptop/issues/4741))
+- Fixed Linux install for RPM installers ([#3774](https://github.com/brave/browser-laptop/issues/3774))([#1445](https://github.com/brave/browser-laptop/issues/1445))
+- Fixed autocomplete on URL bar ([#4731](https://github.com/brave/browser-laptop/issues/4731))([#4360](https://github.com/brave/browser-laptop/issues/4360))([#4573](https://github.com/brave/browser-laptop/issues/4573))
+- Fixed custom filter list to stop using spell check ([#4619](https://github.com/brave/browser-laptop/issues/4619))
+- Fixed numerous issues for Bookmarks Manager ([#4739](https://github.com/brave/browser-laptop/issues/4739))([#4685](https://github.com/brave/browser-laptop/issues/4685))([#4751](https://github.com/brave/browser-laptop/issues/4751))([#2652](https://github.com/brave/browser-laptop/issues/2652))([#4728](https://github.com/brave/browser-laptop/issues/4728))([#1187](https://github.com/brave/browser-laptop/issues/1187))([#1994](https://github.com/brave/browser-laptop/issues/1994))([#4523](https://github.com/brave/browser-laptop/issues/4523))([#4562](https://github.com/brave/browser-laptop/issues/4562))
```

Comment:
#1187 here should be removed as it was just replaced with #4745, which remains open yet.